third_party/sodium/linux/include

```diff
@@ -0,0 +1 @@
+../apple/include
```

Comment:
Spoke offline, I think these symlinks are one weird thing in apple system headers away from an annoying build error.

Reply:
Done. I force-pushed the branch to reset the second to latest commit, which excludes the symlnks.